### DISCUSSION TITLE
Barotrauma steam CMD got broken

### DISCUSSION THREAD

**Discussion by DiscoBo** [FakeFishGames/Barotrauma#15282] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Barotrauma launching in Steam CMD (with anonymously account and mine account with barotraima) returns a error: ERROR! AppId 1026340 launch failed (No subscription). This thing happens since 27.11.24 (I know it because i host barotrauma server every day). 
![image](https://github.com/user-attachments/assets/cc2db901-a868-4d34-974f-e08c29364491)
![image (1)](https://github.com/user-attachments/assets/db77c384-1908-4760-bba5-ba7f66357175)

### Reproduction steps

1. Launch Steam CMD
2. Use commads below:
force_install_dir C:\SteamCMD\barotrauma_server_1
login anonymously
app_run 1026340

P.S.
You can also use your steam account in login

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
ERROR! AppId 1026340 launch failed (No subscription)
```

**Comment by Regalis11** [comment]
Thank you for the report!

It seems that this has to do with some of the changes Steam did last week to how dedicated servers are configured. We have attempted to address this by enabling a new setting that should allow the server to be launched while logged in anonymously, but it doesn't seem to work (so we are suspecting this is an issue at Steam's end). We've contacted Steam's support to help them resolve the issue, but haven't yet heard back from them. 

Sorry for the inconvenience, I hope we can resolve this soon! For the time being, it should be possible to work around the issue by launching the executable directly from the install folder, or by logging into an account before attempting to launch it via SteamCMD.

**Reply by Regalis11** [reply]
Update on this: this is indeed caused by changes at Steam's end, more specifically the discontinuation of their anonymous steamcmd package which previously allowed launching games through steamcmd anonymously. Unfortunately [their documentation](https://partner.steamgames.com/doc/sdk/uploading/distributing_gs) hasn't been updated to explain how the server should be configured now, and we haven't gotten any instructions from their support either yet (although the support person we got in touch with said they'd be looking into it and getting back to us).
